### DISCUSSION TITLE
GitHub Desketop: insufficient memory resources available to process this command

### DISCUSSION THREAD

**Discussion by Oni9834** [community/community#146619] [New to GitHub] [answered]
Hello, yesterday I installed GitHub Desktop, but it's impossible for me to connect my GitHub account to the app, every time a little window opens and tells me “There are insufficient memory resources available to process this command”.
Does anyone have a solution?

**Comment by nishanmudalige** [comment, marked as answer]
There is a similar thread with the same issue: https://github.com/desktop/desktop/issues/15217

The user was able to solve this by deleting a credential.

**Reply by Oni9834** [reply]
It was so simple to fix this and I couldn't find anything... Thank you so much for your help !
